modules/simul/src/main/ui/SimulHome.scala

```diff
@@ -23,7 +23,7 @@ site.StrongSocket.defaultParams.flag = 'simul';
 site.pubsub.on('socket.in.reload', () =>
   fetch('${routes.Simul.homeReload}').then(r => r.text()).then(html => {
   $$('.simul-list__content').html(html);
-  site.contentLoaded();
+  site.pubsub.emit('content-loaded');
```

Comment:
we now have a dozen of hardcoded `'content-loaded'` in the wild, which I guess is okay

Reply:
we could put it into a function somewhere, but i don't think we should be using the site object or index.d.ts as a const string repository.